Controller/Payment/Webhook.php

```diff
@@ -472,4 +537,22 @@ protected function getPostData() : array
         
         return json_decode($request, true);
     }
+
+    protected function getOrderWebhookData($orderId) : array
+    {
+        $collection = $this->objectManagement->get('Magento\Sales\Model\Order')
+        ->getCollection()
```

Comment:
Please follow the chaining functions, alignment formatting rules

Reply:
@nikhilsharma1100 Updated formatting for chaining functions